pycm/pycm_obj.py

```diff
@@ -866,3 +866,87 @@ def to_array(self, normalized=False, one_vs_all=False, class_name=None):
             row = [table[key][i] for i in classes]
             array.append(row)
         return numpy.array(array)
+
+    def combine(self, other):
+        """
+        Return the combine of self Confusion matrix and other one.
+
+        :param other: Other matrix that is going to be combined.
+        :type other: Confusion matrix
+        :return: combine of two matrices as a new Confusion matrix
```

Comment:
Please change it to:
:return: The combination of two matrices as a new confusion matrix

Reply:
Fixed in https://github.com/sepandhaghighi/pycm/pull/344/commits/b5eb3bc08c02b59bd9a452cfa786c0ca9dd412f0